packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -30,6 +32,7 @@ export 'package:flutter/rendering.dart' show SelectionChangedCause;
 typedef SelectionChangedCallback = void Function(TextSelection selection, SelectionChangedCause cause);
 
 const Duration _kCursorBlinkHalfPeriod = Duration(milliseconds: 500);
+const Duration _kCursorBlinkWaitForStart = Duration(milliseconds: 150);
```

Comment:
I'm not sure I could figure out what this is from the name. Add comments.

Reply:
Is it the same amount of time it waits before animating to become opaque as well? While you're here, document the 500ms above too and how they relate.